### PR TITLE
Eslint conf

### PR DESCRIPTION
**Pull request by cbeauchesne**
here we go!

* HTML and JS -> two spaces indent
* JS -> semi !

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -8,8 +8,8 @@ indent_style = spaces
 trim_trailing_whitespace = true
 
 [*.html]
-indent_size = 4
+indent_size = 2
 
 [*.js,*.ts,*.json,*.xml]
 # to be modified soon :)
```
Comment (brunobesson) [inline .editorconfig]: remove the comment, then 🙂

File: .eslintrc.json [hunk 2]
```diff
@@ -36,7 +36,10 @@
     "eqeqeq": "warn",
     "camelcase": "off",
     "no-trailing-spaces": "warn",
-    "semi": "warn",
+    "semi": [
+      "warn",
```
Comment (brunobesson) [inline .eslintrc.json]: for me, it should be an error

File: .editorconfig (hunk 1 above)
Comment (cbeauchesne) [reply .editorconfig]: Done

File: .eslintrc.json (hunk 2 above)
Comment (cbeauchesne) [reply .eslintrc.json]: done

**Pull request merged**